docs/xshareclasses.md

```diff
@@ -345,6 +351,15 @@ case, the VM continues without using shared classes.
 
 :   Disables the storage of classes that are loaded by the bootstrap class loader in the shared classes cache. Often used with the `SharedClassURLFilter` API to control exactly which classes are cached. For more information about shared class filtering, see [Using the SharedClassHelper API](https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.vm.80.doc/docs/shrc_pd_helper_class.html).
 
+### `noCheckURLTimestamps`
+
+        -Xshareclasses:noCheckURLTimestamps
+
+:   Timestamps of `jar` or `zip` files are checked only when they are added to a class loader and used for the first time to look up a class. This is the default
```

Comment:
`CheckURLTimeStamps` should have a lower case `c`

Reply:
Fixed. Seems odd - do you have rules for these?

Reply:
Right or wrong, all the `-Xshareclasses:` sub-options start with lower case.